<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>net2brain</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(56, 58, 89);
	color: rgb(235, 252, 255);
	border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(56, 58, 89);
	color: rgb(235, 252, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_sub">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 121, 199);</string>
     </property>
     <property name="text">
      <string>Select your action</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>450</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(140, 164, 230);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_feats">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(140, 164, 230);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string>Generate Features</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_eval">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(140, 164, 230);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Evaluation</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_title">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 121, 199);
</string>
     </property>
     <property name="text">
      <string>&lt;strong&gt;net&lt;/strong&gt;2&lt;strong&gt;brain&lt;/strong&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rdm">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>330</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(140, 164, 230);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Create RDMs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
